code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -125,3 +125,9 @@
 	desc = "Allows you to fly by using your wings."
 	cost = 2 //Some in game value.
 	var_changes = list("can_fly" = 1)
+
+/datum/trait/hardfeet
+	name = "Hard Feet"
+	desc = "Do you feel like your nice clawed, scaled, hooved, armored, or otherwise just hella calloused feet need more freedom without the fear of glass shards?"
```

Comment:
Description doesn't properly describe what the trait does, nor does it align with existing trait descriptions.